README.md

```diff
@@ -1,2 +1,86 @@
-# the-project
-The Project is a sample project we ask prospective consultants to complete as part of their interview process. 
+# The Project
+
+## Introduction
+
+`The Project` is a self-guided application project for all prospective Lofty consultants. We believe that an engineer's merit is not always described by their resumé. Sometimes a great engineer hasn't had an opportunity to prove themselves yet or maybe they're just uncomfortable in interviews and don't end up giving their true ability the chance to shine in an stressful interview. `The Project` helps us determine a potential consultant's true ability outside of the black and white confines of their resumé and sometimes those awkward face-to-face interviews.
+
+`The Project` will explore your capability as a Python developer using our preferred tooling. We don't expect all applicants to have proficiency in all of the technologies specified in this document.  Lofty Labs Consultants often work independently with only a small amount of direct supervision and often work directly on projects built by other engineering teams as well. Sometimes this introduces new systems to us that we haven't used before. This project is meant to expose you to the same situation wherein maybe you haven't used all of the technologies or tooling that we have specified in the requirements. Completion of `The Project` will show that even if you don't have experience with all of the specifications listed, you're a dedicated and passionate engineer that is able to pick up new systems and technologies at an at least basic level. The ability to adapt to a new and possibly unknown technology is a core competency of all Lofty Engineering Consultants.
```

Comment:
>Lofty Labs Consultants often work independently with only a small amount of direct supervision and often work directly on projects built by other engineering teams as well.

Lofty Labs Consultants often work independently with only a small amount of direct supervision and, frequently, on projects built by other engineering teams.